cli/flox/src/commands/install.rs

```diff
@@ -218,20 +233,97 @@ impl Install {
         );
 
         match err {
+            EnvironmentError::Core(CoreEnvironmentError::LockedManifest(
+                LockedManifestError::ResolutionFailed(failures),
+            )) if failures.0.iter().all(|f| {
+                matches!(f, ResolutionFailure::PackageUnavailableOnSomeSystems { .. })
+            }) =>
+            {
+                let mut packages = packages
+                    .iter()
+                    .cloned()
+                    .map(|p| (p.id().to_string(), p))
+                    .collect::<HashMap<_, _>>();
+
+                for failure in &failures.0 {
+                    let ResolutionFailure::PackageUnavailableOnSomeSystems {
+                        catalog_message:
+                            MsgAttrPathNotFoundNotFoundForAllSystems {
+                                install_id,
+                                valid_systems,
+                                ..
+                            },
+                        invalid_systems: _,
+                    } = failure
+                    else {
+                        unreachable!("already checked that these failures are 'package unavailable on some systems'")
+                    };
+
+                    let Some(package_to_install) = packages.get_mut(install_id) else {
+                        warn!(install_id, "resolution failure for non-existent package");
```

Comment:
Actually this is a cli impl so we can also pretty print with `message::warning`

Reply:
I think making it ugly is fine because we don't expect it to happen. And if it does it means there's a bug

Reply:
yeah i was thinking the same, besides i think warn! and error! end up in sentry iirc